docs/_lab/03-running-commands/index.md

```diff
@@ -4,98 +4,98 @@ difficulty: Basic
 time: Approximately 5 minutes
 ---
 
-You can use Bolt to run arbitrary commands on a set of remote hosts. Let's see that in practice before we move on to more advanced features. Choose the exercise based on the operating system of your test nodes.
+You can use Bolt to run arbitrary commands on a set of remote hosts. Let's see that in practice before we move on to more advanced features. Choose the exercise based on the operating system of your test targets.
 
-- [Running Shell Commands on Linux Nodes](#running-shell-commands-on-linux-nodes)
-- [Running PowerShell Commands on Windows Nodes](#running-powershell-commands-on-windows-nodes)
+- [Running Shell Commands on Linux Targets](#running-shell-commands-on-linux-targets)
+- [Running PowerShell Commands on Windows Targets](#running-powershell-commands-on-windows-targets)
 
 ## Prerequisites
 Complete the following before you start this lesson:
 
 - [Installing Bolt](../01-installing-bolt)
-- [Setting Up Test Nodes](../02-acquiring-nodes)
+- [Setting Up Test Targets](../02-acquiring-targets)
 
-## Running Shell Commands on Linux Nodes
+## Running Shell Commands on Linux Targets
 
 Bolt by default uses SSH for transport. If you can connect to systems remotely, you can use Bolt to run shell commands. It reuses your existing SSH configuration for authentication, which is typically provided in `~/.ssh/config`.
 
-To run a command against a remote Linux node, use the following command syntax:
+To run a command against a remote Linux target, use the following command syntax:
 ```shell
-bolt command run <command> --nodes <nodes>
+bolt command run <command> --targets <targets>
 ```
 
-To run a command against a remote node using a username and password rather than keys use the following syntax:
+To run a command against a remote target using a username and password rather than keys use the following syntax:
 ```shell
-bolt command run <command> --nodes <nodes> --user <user> --password <password>
+bolt command run <command> --targets <targets> --user <user> --password <password>
 ```
 
-Run the `uptime` command to view how long the system has been running. If you are using existing nodes on your system, replace `node1` with the address for your node.
+Run the `uptime` command to view how long the system has been running. If you are using existing targets on your system, replace `target1` with the address for your target.
 
 ```shell
-bolt command run uptime --nodes node1
+bolt command run uptime --targets target1
 ```
 
 The result:
 ```
-Started on node1...
-Finished on node1:
+Started on target1...
+Finished on target1:
   STDOUT:
      22:42:18 up 16 min,  0 users,  load average: 0.00, 0.01, 0.03
-Successful on 1 node: node1
-Ran on 1 node in 0.42 seconds
+Successful on 1 target: target1
+Ran on 1 target in 0.42 seconds
```

Comment:
I don't think we've changed this output yet...https://github.com/puppetlabs/bolt/blob/master/lib/bolt/outputter/human.rb#L170 Should that be part of this change?